### PR TITLE
[core] #318 clone repos

### PR DESCRIPTION
**Pull request by PiyushMishra**

### CODE REVIEW

File: core/src/main/scala/com/kodebeagle/crawler/ZipBasicParser.scala [hunk 1]
```diff
@@ -87,3 +86,46 @@ object ZipBasicParser extends Logger {
   def listAllFiles(dir: String): Array[File] = new File(dir).listFiles
 }
 
+object ZipUtil extends Logger {
+
+  def createZip(directoryPath: String, zipPath: String) {
+    try {
+      Option(new FileOutputStream(new File(zipPath))).foreach { fOut =>
```
Comment (ScrapCodes) [inline core/src/main/scala/com/kodebeagle/crawler/ZipBasicParser.scala]: What is the use of Option here ?

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/MainWindow.java [hunk 2]
```diff
@@ -23,13 +23,11 @@
 import com.imaginea.kodebeagle.action.RefreshAction;
 import com.imaginea.kodebeagle.object.WindowObjects;
 import com.imaginea.kodebeagle.util.WindowEditorOps;
-import com.intellij.ide.actions.ActivateToolWindowAction;
```
Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/MainWindow.java]: Stray changes ?

Comment (PiyushMishra) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/MainWindow.java]: Strange, Looking into it.

**Comment by PiyushMishra** [comment]
LGTM

**Pull request closed without merging**